src/commands.cpp

```diff
@@ -1429,8 +1429,55 @@ void CommandLogout::procresult()
     app->logout_result(e);
 }
 
+CommandPrelogin::CommandPrelogin(MegaClient* client, const char* email)
+{
+    cmd("us0");
+    arg("user", email);
+
+    this->email = email;
+    tag = client->reqtag;
+}
+
+void CommandPrelogin::procresult()
+{
+    if (client->json.isnumeric())
+    {
+        return client->app->prelogin_result(0, NULL, NULL, (error)client->json.getint());
+    }
+
+    int v = 1;
```

Comment:
Maybe we can default it to `0`. Then, in the `EOO`, we can really check whether a `v=1` has been received. Otherwise, return error.